src/App.jsx

```diff
@@ -14,45 +15,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_ALPHABETICALLY = 'alphabetically';
+const SORT_FIELD_LENGTH = 'length';
+
+export const App = () => {
+  let goods = [...goodsFromServer];
+  const [sortField, setSortField] = useState('');
+  const [reverseList, setReverseList] = useState(false);
+
+  switch (sortField) {
+    case SORT_FIELD_ALPHABETICALLY:
+      goods = goods.sort((a, b) => a.localeCompare(b));
+      break;
+    case SORT_FIELD_LENGTH:
+      goods = goods.sort((a, b) => a.length - b.length);
+      break;
+    default:
+      break;
+  }
+
+  if (reverseList) {
+    goods = goods.reverse();
+  }
```

Comment:
Move this logic to a separate function outside the component to keep the component clean and easy to understand